lib/new_relic/agent/instrumentation/stripe_subscriber.rb

```diff
@@ -0,0 +1,71 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    module Instrumentation
+      class StripeSubscriber
+        DEFAULT_DESTINATIONS = AttributeFilter::DST_SPAN_EVENTS
+        EVENT_ATTRIBUTES = %i[http_status method num_retries path request_id].freeze
+        ATTRIBUTE_NAMESPACE = 'stripe.user_data'
+        ATTRIBUTE_FILTER_TYPES = %i[include exclude].freeze
+
```

Comment:
I think some of the methods in this class could be made private. `start_segment` and `finish_segment` should probably stay public, but I think the others could be made private.

Reply:
Nice suggestion.. done! https://github.com/newrelic/newrelic-ruby-agent/pull/2180/commits/c59f04c4a06e2c6e404ec8f9daf89be8c6dbbfa7